README.md

```diff
@@ -1,493 +1,69 @@
 # Trusted Service Identity (TSI)
```

Comment:
Also https://github.com/IBM/trusted-service-identity/tree/tornjak-spire/examples/vault is no longer valid with SPIRE anymore. Think we can just remove this since SPIRE setup with vault is different (using their x509 plugin)

Reply:
`tsi-version.txt` is this still used?